Dockerfile

```diff
@@ -0,0 +1,58 @@
+# The official python:2.7 image no longer receives automatic rebuilds (it's
+# a year old as of October, 2021), so use the latest LTS release of Ubuntu
+# that includes Python 2.7 instead.
+FROM ubuntu:20.04
+
+# Install packages needed to run your application (not build deps).
+RUN set -ex \
+    && RUN_DEPS=" \
+    ca-certificates \
+    git \
+    libpq5 \
+    make \
+    postgresql-client \
+    python2.7 \
+    " \
+    && apt-get update && apt-get install -y --no-install-recommends $RUN_DEPS \
```

Comment:
I'd avoid the variable definition and inline the dependencies. But that's a personal preference.